src/NHSD.BuyingCatalogue.Ordering.Api/Models/CreateOrderItemBaseModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.ComponentModel.DataAnnotations;
+using NHSD.BuyingCatalogue.Ordering.Api.Attributes;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public abstract class CreateOrderItemBaseModel
+    {
+        [Required(ErrorMessage = "ServiceRecipientRequired")]
+        public ServiceRecipientModel ServiceRecipient { get; set; }
+
+        [Required(ErrorMessage = "ProvisioningTypeRequired")]
+        public string ProvisioningType { get; set; }
+
+        [Required(ErrorMessage = "TypeRequired")]
+        public string Type { get; set; }
+
+        [Required(ErrorMessage = "CurrencyCodeRequired")]
+        public string CurrencyCode { get; set; }
+
+        [Required(ErrorMessage = "ItemUnitRequired")]
+        public ItemUnitModel ItemUnit { get; set; }
+
+        [Required(ErrorMessage = "QuantityRequired")]
+        [Limit(1, LimitType.Minimum, ErrorMessage = "QuantityGreaterThanZero")]
+        [Limit(int.MaxValue - 1, LimitType.Maximum, ErrorMessage = "QuantityLessThanMax")]
+        public int? Quantity { get; set; }
+
+        public string EstimationPeriod { get; set; }
+
+        [Required(ErrorMessage = "PriceRequired")]
+        [Limit(0d, LimitType.Minimum, ErrorMessage = "PriceGreaterThanOrEqualToZero")]
```

Comment:
'd' is for a double, 'm' would be consistent with a decimal here, though suppose it doesn't matter.

Reply:
I've switched this to the decimal type, the same as the Maximum. Clears things up a bit